libr/core/graph.c

```diff
@@ -3307,13 +3307,12 @@ R_API void r_agraph_reset(RAGraph *g) {
 	agraph_free_nodes (g);
 	r_agraph_set_title (g, NULL);
 	sdb_reset (g->db);
-	r_list_free (g->edges);
+	r_list_purge (g->edges);
```

Comment:
purge doesnt deletes the edges, just the container, isnt that leaking?

Reply:
I thought I could fix a graph leak, but it turned out to be much harder.
I just marked the `g->nodes` place as XXX.

This commit fix a minor leak issue in cmd.c

Reply:
g->edges is r_list_free (g->edges); and r_list_new latter, `r_list_purge` is better.